committee-steering/governance/README.md

```diff
@@ -19,6 +19,23 @@ Specific attention has been given to:
 - Process for agreeing upon technical decisions
 - Process for ensuring technical assets remain healthy
 
+## SIG Charter approval process
+
+When introducing a SIG charter or modification of a charter the following process should be used:
+
+- Identify a small set of owners from the SIG to drive the changes.
+  Most typically this will be the SIG chairs.
+- Work with the steering committee to gain approval.
+  This can simply be submitting a PR and sending mail to [steering@kubernetes.io].
+  If more substantial changes are desired it is advisable to socialize those before drafting a PR.
```

Comment:
I'd expect the SIG to have consensus before asking the SC to review the changes.

Reply:
> I'd expect the SIG to have consensus before asking the SC to review the changes.

Or to have deadlocked?

Reply:
Reordered to make it more clear. (pushing soon).